Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.access.redhat.com/ubi8/ubi
+
+ENV node_version=v12.18.2
```

Comment:
`ENV` by `ARG` since `node_version` is not needed by the bot

Reply:
We are installing node on the image, so extracted it here.